openpaas-james/apps/distributed-ldap/src/main/java/com/linagora/openpaas/james/app/DistributedLdapServer.java

```diff
@@ -58,7 +60,7 @@
                 new RabbitMailQueueRoutesModule(),
                 new RabbitMQEventBusModule(),
                 new DistributedTaskSerializationModule()))
-        .with(new LdapUsersRepositoryModule(), new CustomMethodModule());
```

Comment:
My guess is here we should even probably delete the distributedLdapServer, and let the LDAPModuleChooser do its job? If yes, probably other PR? And how should we organize that?

Reply:
> My guess is here we should even probably delete the distributedLdapServer, and let the LDAPModuleChooser do its job?

Yes

> If yes, probably other PR?

Ok for me.

> And how should we organize that?

Just drop the distributedLdap server main, keep the test as is with the distributed server configured. For the deployment test we likely should use a DockerFile to customize the usersrepository.xml file.

I'm in favor of dropping the JMAP integration tests for LDAP.

Reply:
https://github.com/linagora/openpaas-james/issues/56